Sources/Security/VerificationResult.swift

```diff
@@ -83,14 +83,11 @@ extension VerificationResult {
         case (.verified, .notRequested): return .notRequested
         case (.verified, .failed): return .failed
 
-        // These shouldn't happen because `ETagManager` will ignore not verified cached responses
-        // if verification is enabled.
         case (.notRequested, .verified): return .notRequested
```

Comment:
Should we also update this case to `verified`? Though in that case, we should just pick the responseResult always instead of this switch.

Reply:
Yeah I'll update it, that makes sense.
I think we might want to keep the switch and add a third argument: the mode. Because I predict we'll implement this differently for `.enforced`.